src/landpatterns/numbering.stanza

```diff
@@ -183,6 +183,16 @@ numbering scheme. For example - a `col-offset = 1` would construct:
     8   12
   ```
 
+@param col-skip Value to divide the raw column value by. This parameter
+is useful when working with staggered grids where a pad is active every
+other column. When this value is greater than 1 - then the numbering scheme
+will accept twice as many columns as normal.
```

Comment:
Perhaps a small example would help solidify the explanation here?

I'm assuming "will accept twice as many as normal" means that the numbering scheme would normally not accept values higher than `num-cols` or `num-rows` but if a skip parameter is passed in, then it'll adjust for that and know to allow up to `<skip-value> * num-cols/rows`

Reply:
Ya - I agree that a diagram and an example would be helpful.

> I'm assuming "will accept twice as many as normal" means that the numbering scheme would normally not accept values higher than num-cols or num-rows but if a skip parameter is passed in, then it'll adjust for that and know to allow up to <skip-value> * num-cols/rows

Yes